--- khtml-5.32.0/src/khtml_factory.h.omv~	2017-03-07 01:11:40.823386417 +0100
+++ khtml-5.32.0/src/khtml_factory.h	2017-03-07 01:11:45.455425446 +0100
@@ -28,7 +28,7 @@
 class KHTMLFactory : public KPluginFactory
 {
     Q_OBJECT
-    Q_PLUGIN_METADATA(IID "org.kde.KPluginFactory" FILE "")
+    Q_PLUGIN_METADATA(IID "org.kde.KPluginFactory")
     Q_INTERFACES(KPluginFactory)
 public:
     KHTMLFactory();
